README.rst

```diff
@@ -159,7 +163,9 @@ Rather, you can just import ``mayavi.mlab`` and pass it to any figure list that
 
 Installation for developers
 ---------------------------
-To install from github, just ``git clone https://github.com/jmfranck/pyspecdata.git``. Then switch over to the anaconda prompt and move to the directory where setup.py lives (ex. ``git/pyspecdata/``),
+**Warning** Before running the installation for developers, you must first check that the output of ``conda info`` on your git bash terminal matches the output of your anaconda prompt. If these do not match you will have to make the conda environment accessible to all users. 
+
+Once this is checked, to install pySpecData from github, just ``git clone https://github.com/jmfranck/pyspecdata.git``. Then switch over to the anaconda prompt and move to the directory where setup.py lives (ex. ``git/pyspecdata/``),
```

Comment:
this is called the "root directory of the repository"

Reply:
don't give this example -- no guarantee that other people organize their personal directory like you